packages/react-mutation-mapper/src/util/MutationTypeUtils.ts

```diff
@@ -61,41 +69,66 @@ export function mutationTypeSort(
 }
 
 export function getColorForProteinImpactType(
-    mutations: Partial<Mutation>[],
+    mutations: Partial<Mutation & { putativeDriver: boolean }>[],
```

Comment:
`putativeDriver` field is cBioPortal specific, right? Instead of adding a new field to the model, should we make it more generic by passing another function to this function? 

```suggestion
    mutations: Partial<Mutation>[],
    isPutativeDriver(mutation: Partial<Mutation>) => boolean = () => false,
```

On the cBioPortal side we can pass a function like this:
```JavaScript
m => m.putativeDriver
```

Reply:
sure, thats a good idea